src/lib/styles/theme/components/button.ts

```diff
@@ -161,6 +161,7 @@ export const Button: ComponentStyleConfig = {
       basic: {
         border: borderDefault,
         borderColor: gray400,
+        background: "background.main",
```

Comment:
hmm this doesn't break components in other pages right?

Reply:
Nope, the outline variant normally contain the same background color as background main color and has border color as popup color instead. However, we didn't notice about we forgot to add the "real" background color to the button variant, since it look the same. In this case, we prevent the overlap of others layout under outline button variant.